src/js/_enqueues/wp/heartbeat.js

```diff
@@ -733,35 +734,28 @@
 				oldInterval = settings.tempInterval ? settings.tempInterval : settings.mainInterval;
 
 			if ( speed ) {
-				switch ( speed ) {
-					case 'fast':
-					case 5:
-						newInterval = 5000;
-						break;
-					case 15:
-						newInterval = 15000;
-						break;
-					case 30:
-						newInterval = 30000;
-						break;
-					case 60:
-						newInterval = 60000;
-						break;
-					case 120:
-						newInterval = 120000;
-						break;
-					case 'long-polling':
-						// Allow long polling (experimental).
-						settings.mainInterval = 0;
-						return 0;
-					default:
+				if ( 'fast' === speed ) {
+					// Special case, see below.
+					newInterval = 5000;
+				} else if ( 'long-polling' === speed ) {
```

Comment:
Should this also check for `speed === 0` as an equivalent for long polling so it hits the early return? 

```suggestion
				} else if ( 'long-polling' === speed ) {
```

Reply:
Sure but that would be an enhancement? I mean, long-polling is still "experimental" after 11 years. Maybe it would work better now, but still not sure if it is worth using it. There seem to be few newer and better options. Perhaps keeping this setting as-is would be better?

Reply:
> Maybe it would work better now, but still not sure if it is worth using it. There seem to be few newer and better options. Perhaps keeping this setting as-is would be better?

Cool, works for me.